docs/cars.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python3
+import argparse
+from collections import defaultdict, namedtuple
+from enum import Enum
+import jinja2
+import os
+from sortedcontainers import SortedList
+from typing import Dict
+
+from common.basedir import BASEDIR
+from common.params import Params
+from selfdrive.car.car_helpers import interfaces, get_interface_attr
+from selfdrive.car.chrysler.values import CAR as CHRYSLER
+from selfdrive.car.gm.values import CAR as GM
+from selfdrive.car.honda.values import CAR as HONDA
+from selfdrive.car.hyundai.radar_interface import RADAR_START_ADDR as HKG_RADAR_START_ADDR
+from selfdrive.car.hyundai.values import CAR as HYUNDAI
+from selfdrive.car.toyota.values import CAR as TOYOTA
+from selfdrive.car.volkswagen.values import CAR as VOLKSWAGEN
+from selfdrive.test.test_routes import non_tested_cars
+
+
+class Tier(Enum):
+  GOLD = "Gold"
+  SILVER = "Silver"
+  BRONZE = "Bronze"
+
+
+class Column(Enum):
+  MAKE = "Make"
+  MODEL = "Model"
+  PACKAGE = "Supported Package"
+  LONGITUDINAL = "openpilot Longitudinal"
+  FSR_LONGITUDINAL = "FSR Longitudinal"
+  FSR_STEERING = "FSR Steering"
+  STEERING_TORQUE = "Steering Torque"
+  SUPPORTED = "Actively Maintained"
+
+
+StarColumns = list(Column)[3:]
+CarException = namedtuple("CarException", ["cars", "text", "column", "star"], defaults=[None])
+
+
+def get_star_icon(variant):
+  return '<img src="assets/icon-star-{}.png" width="22" />'.format(variant)
+
+
+def get_exceptions(CP) -> Dict[Column, CarException]:
+  exceptions = {}
+  for car_exception in CAR_EXCEPTIONS:
+    if CP.carFingerprint in car_exception.cars:
+      exceptions[car_exception.column] = car_exception
+  return exceptions
+
+
+CARS_MD_OUT = os.path.join(BASEDIR, "docs", "CARS_generated.md")
+
+# TODO: which other makes?
+MAKES_GOOD_STEERING_TORQUE = ["toyota", "hyundai", "volkswagen"]
+CAR_EXCEPTIONS = [
```

Comment:
call this footnotes and set this from values.py

Reply:
How's that now?